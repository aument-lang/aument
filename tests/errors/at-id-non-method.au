def a() {
    @a = 1;
}