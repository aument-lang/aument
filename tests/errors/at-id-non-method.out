parser error(8) in -: this variable can only be accessed in a method
2 |     @a = 1;
        ^^
